src/test/java/com/github/tomakehurst/wiremock/client/ResponseDefinitionBuilderTest.java

```diff
@@ -156,4 +170,54 @@ void proxyResponseDefinitionWithExtraInformationIsInResponseDefinitionWithBinary
         equalTo(new HttpHeaders(List.of(new HttpHeader("header", "value")))));
     assertThat(proxyDefinition.getProxyUrlPrefixToRemove(), equalTo("/remove"));
   }
+
+  @Test
+  void responseDefinitionBuilderWithGzipDisabled() {
+    ResponseDefinition responseDefinition =
+        ResponseDefinitionBuilder.responseDefinition().withGzipDisabled(true).build();
+
+    assertNotNull(responseDefinition);
+    assertEquals(
+        "none", responseDefinition.getHeaders().getHeader(CONTENT_ENCODING).getValues().get(0));
+  }
+
+  @Test
+  void wireMockServerWithStubForWithGzipDisabledTrue() {
```

Comment:
Suggest we move this into an existing acceptance test class or create a new one, since it's that style of test.

Reply:
done